### PR TITLE
Acceptance strategy

### PR DESCRIPTION
**Pull request by tomvand**
First version of the acceptance strategy. The acceptance strategy ACcombi is used, this is described in "Baarslag, T. Hindriks, K. and Jonker, C. M. (2013), “Acceptance Conditions in Automated Negotiation”, Complex Automated Negotiations: Theories, Models, and Software Competitions, 435, pp. 95-111. http://ii.tudelft.nl/sites/default/files/Acceptance%20Conditions%20in%20Automated%20Negotiation-pub.pdf".

### CODE REVIEW

**Comment by alexwalterbos** [comment]
Aww conflicts :( Code looks good, plenty of comments and good readability!

**Comment by tomvand** [comment]
It looks like the acceptance strategy code is removed after merging master into AcceptanceStrategy?

**Comment by alexwalterbos** [comment]
Then I've missed something. I'll fix it, should be fine in a couple of minutes!

**Comment by alexwalterbos** [comment]
@tomvand should be good now! Final check please =D

**Comment by tomvand** [comment]
Yes it's looking fine now, thanks :) Merging...

File: src/negotiator/group1/Group1_AS.java [hunk 1]
```diff
@@ -1,14 +1,144 @@
...
-
+	
+	// Context in which the negotiation takes place.
+	NegotiationSession negotiationSession;
+	OfferingStrategy offeringStrategy;
+	
+	// Parameters for the acceptance strategy.
+	double deadline = 0.99;
+	double a = 1.0;
+	double b = 0.0;
```
Comment (alexwalterbos) [inline src/negotiator/group1/Group1_AS.java:21]: I had missed these, @tomvand. Could you use a bit more explanatory naming and/or comments?

Comment (alexwalterbos) [inline src/negotiator/group1/Group1_AS.java:21]: My fucking god, these names are prescribed in the assignment?!?! Why the hell do people teaching Master students not understand the concept of general coding guidelines?! 
Well, anyway, I understand `a` resembles some factor of the utility, and `b` an addition to this value. I suggest the naming `utilityFactor` and `utilityAddition`, or something similar. Sorry for the earlier comment, obviously not your fault.

Comment (tomvand) [inline src/negotiator/group1/Group1_AS.java:21]: np, I agree that `a` and `b` are terribly uninformative names. Will fix this for the final version :)

**Pull request merged**